Problem037/C++/solution_1.cpp

```diff
@@ -0,0 +1,49 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+const int much = 1000000;
+bool is_prime[much];
+
+void sieve(){
+	memset (is_prime, true, sizeof(is_prime));
+	is_prime[0] = is_prime [1] = false;
```

Comment:
Remove the space character before the `[...]`. The first one is quite right with `is_prime[0]` but the latter one is not so.

```suggestion
      is_prime[0] = is_prime[1] = false;
```